samples/test/README.md

```diff
@@ -1 +1,19 @@
-The files here are used for local development only.
\ No newline at end of file
+The files here are used for local development only.
+
+To test `msvc-oci` on Kind, the cluster must be configured with the `ImageVolume` featuer enabled, otherwise the OCI volume will not get created in child deployments. Create the cluster with the following command.
+
+```
+kind create cluster --config samples/test/kind-config.yaml
```

Comment:
I think we should create the cluster with the desired Kubernetes version

Reply:
Done.